homeassistant/components/growatt_server/sensor.py

```diff
@@ -12,12 +12,10 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.components.sensor import PLATFORM_SCHEMA
```

Comment:
The config schema should be defined in the component instead, `CONFIG_SCHEMA`. It shouldn't extend `PLATFORM_SCHEMA`.